ParseTwitterUtils/ParseTwitterUtils.xcodeproj/project.pbxproj

```diff
@@ -947,6 +951,8 @@
 			baseConfigurationReference = F51535531B57453700C49F56 /* ParseTwitterUtils-iOS.xcconfig */;
 			buildSettings = {
 				IPHONEOS_DEPLOYMENT_TARGET = 8.0;
+				SUPPORTS_MACCATALYST = YES;
```

Comment:
The rest of the project files say NO to Catalyst support. I assume this is a mistake?